app/src/main/java/com/android/sample/model/recipe/RecipeBuilder.kt

```diff
@@ -24,36 +24,36 @@ class RecipeBuilder {
    *
    * @param idMeal The ID of the recipe.
    */
-  fun setId(idMeal: String) = apply { this.idMeal = idMeal }
+  fun setId(idMeal: String) = apply { this.uid = idMeal }
 
   /**
    * Sets the name of the recipe.
    *
    * @param strMeal The name of the recipe.
    */
-  fun setName(strMeal: String) = apply { this.strMeal = strMeal }
+  fun setName(strMeal: String) = apply { this.name = strMeal }
 
   /**
    * Sets the category of the recipe.
    *
    * @param strCategory The category of the recipe.
    */
-  fun setCategory(strCategory: String) = apply { this.strCategory = strCategory }
+  fun setCategory(strCategory: String) = apply { this.category = strCategory }
 
   /**
    * Sets the area of the recipe.
    *
    * @param strArea The area of the recipe.
    */
-  fun setArea(strArea: String) = apply { this.strArea = strArea }
+  fun setArea(strArea: String) = apply { this.origin = strArea }
```

Comment:
I feel here the parameter name should definitely be refactored to origin as Area and origin seem fairly different.